fhttp/http.go

```diff
@@ -1151,13 +1156,23 @@ func EchoHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-var payload = make([]byte, MaxPayloadSize)
-
 func init() {
+	if len(payload) == 0 {
```

Comment:
that will always be true as init runs before flag parsing; it wasn't what I meant (even if I can see how it can be interpreted that way)
it was a) call the function from init - or even better and optional - not ChangeMaxPayloadSize twice